snapcraft/plugins/python3.py

```diff
@@ -166,10 +167,22 @@ def build(self):
              '--prefix={}/usr'.format(self.installdir),
              '--root={}'.format(self.installdir)], cwd=self.builddir)
 
+        def remove_func(dirpath, files):
+            print(dirpath, files)
+            endswith_pyc = (x.endswith('.pyc') for x in files)
+            return dirpath.endswith('__pycache__') and all(endswith_pyc)
+
+        file_utils.remove_dirs(self.installdir, remove_func)
+
         # Fix all shebangs to use the in-snap python.
-        common.replace_in_file(self.installdir, re.compile(r''),
-                               re.compile(r'#!.*python'),
-                               r'#!/usr/bin/env python')
+        file_utils.replace_in_file(self.installdir, re.compile(r''),
+                                   re.compile(r'#!.*python'),
+                                   r'#!/usr/bin/env python')
+
+    def snap_fileset(self):
+        fileset = super().snap_fileset()
+        fileset.append('-usr/bin/pip*')
+        return fileset
 
     @property
```

Comment:
I think the properties go between the __init__ and the methods. But I'm not sure where did I see this guideline.